ush/config_defaults.yaml

```diff
@@ -1632,48 +1654,57 @@ task_get_extrn_lbcs:
-  #
-  #-----------------------------------------------------------------------
-  #
-  # EXTRN_MDL_SYSBASEDIR_LBCS:
-  # Same as EXTRN_MDL_SYSBASEDIR_ICS but for LBCs.
-  #
-  # Note that this variable must be defined as a null string here so that if it is 
-  # specified by the user in the experiment configuration file, it remains 
-  # set to those values, and if not, it gets set to machine-dependent 
-  # values.
-  #
-  #-----------------------------------------------------------------------
-  #
-  EXTRN_MDL_SYSBASEDIR_LBCS: ''
-  #
-  #-----------------------------------------------------------------------
-  #
-  # User-staged external model directories and files.  Definitions:
-  #
-  # USE_USER_STAGED_EXTRN_FILES:
-  # Analogous to USE_USER_STAGED_EXTRN_FILES in ICS but for LBCs.
-  #
-  # EXTRN_MDL_SOURCE_BASEDIR_LBCS:
-  # Analogous to EXTRN_MDL_SOURCE_BASEDIR_ICS but for LBCs instead of ICs.
-  #
-  # EXTRN_MDL_FILES_LBCS:
-  # Analogous to EXTRN_MDL_FILES_ICS but for LBCs instead of ICs.
-  #
-  #-----------------------------------------------------------------------
-  #
-  USE_USER_STAGED_EXTRN_FILES: false
-  EXTRN_MDL_SOURCE_BASEDIR_LBCS: ""
-  EXTRN_MDL_FILES_LBCS: ""
+  envars:
+    EXTRN_MDL_NAME_LBCS: "FV3GFS"
+    LBC_SPEC_INTVL_HRS: 6
+    EXTRN_MDL_LBCS_OFFSET_HRS: '{{ 3 if task_get_extrn_lbcs.envars.EXTRN_MDL_NAME_LBCS == "RAP" else 0 }}'
+    FV3GFS_FILE_FMT_LBCS: "nemsio"
+    #
+    #-----------------------------------------------------------------------
+    #
+    # EXTRN_MDL_SYSBASEDIR_LBCS:
+    # Same as EXTRN_MDL_SYSBASEDIR_ICS but for LBCs.
+    #
+    # Note that this variable must be defined as a null string here so that if it is 
+    # specified by the user in the experiment configuration file, it remains 
+    # set to those values, and if not, it gets set to machine-dependent 
+    # values.
+    #
+    #-----------------------------------------------------------------------
+    #
+    EXTRN_MDL_SYSBASEDIR_LBCS: ''
+    #
+    #-----------------------------------------------------------------------
+    #
+    # User-staged external model directories and files.  Definitions:
+    #
+    # USE_USER_STAGED_EXTRN_FILES:
+    # Analogous to USE_USER_STAGED_EXTRN_FILES in ICS but for LBCs.
+    #
+    # EXTRN_MDL_SOURCE_BASEDIR_LBCS:
+    # Analogous to EXTRN_MDL_SOURCE_BASEDIR_ICS but for LBCs instead of ICs.
+    #
+    # EXTRN_MDL_FILES_LBCS:
+    # Analogous to EXTRN_MDL_FILES_ICS but for LBCs instead of ICs.
+    #
+    #-----------------------------------------------------------------------
+    #
+    USE_USER_STAGED_EXTRN_FILES: false
+    EXTRN_MDL_SOURCE_BASEDIR_LBCS: ""
+    EXTRN_MDL_FILES_LBCS: ""
 
 #----------------------------
 # MAKE ICS config parameters
 #-----------------------------
 task_make_ics:
+
+  chgres_cube:
+    execution: &chgres_cube_execution
+      batchargs:
+        walltime: 00:30:00
+        threads: 1
+        nodes: 4
+        tasks_per_node: 12
```

Comment:
I tried to follow what I expect will come from @WeirAE's contribution of make_ics/make_lbcs.